src/components/JobCard/JobCard.tsx

```diff
@@ -1,8 +1,53 @@
-import React from "react";
-import { Text } from "react-native";
+import { Text, View, Pressable, Modal } from "react-native";
+import styles from './CardStyles'
+import React, {useState} from "react";
 
-const JobCard = () => {
-  return <Text>hello</Text>;
-};
+const JobCard = ({title, description, employer, hours, salary, contact}:{title: string; description: string}) => {
```

Comment:
you should make a new props object here for the props i.e. `JobCardProps`. see this component as an example: https://github.com/calblueprint/sunday-friends-mobile/blob/main/src/components/ProfileEditModal/ProfileEditModal.tsx